src/features/CarePackages/DayCare/DayCare.js

```diff
@@ -6,46 +6,95 @@ import CareTitle from "../components/CareTitle";
 import TextArea from "../../components/TextArea";
 import { days } from "../../components/daysData";
 import Checkbox from "../../components/Checkbox";
-import { useState } from "react";
+import { useEffect, useState } from "react";
 import RadioButton, { yesNoValues } from "../../components/RadioButton";
 import {
-  getTermTimeConsiderationOptions,
+  createDayCarePackage,
   getOpportunitiesLengthOptions,
-  getOpportunitiesTimePerMonthOptions,
+  getOpportunityTimesPerMonthOptions,
+  getTermTimeConsiderationOptions
 } from "../../../api/CarePackages/DayCareApi";
 import DayCareOpportunities from "./components/DayCareOpportunities";
+import { Button } from '../../components/Button';
+import { CARE_PACKAGE } from '../../../routes/RouteConstants';
+import TitleHeader from '../../components/TitleHeader';
+import DayCareSummary from './components/DayCareSummary';
+
+const DayCare = ({history}) => {
+  const isTrueSet = (myValue) => (myValue === 'true');
+  const checkFixedPeriod = (myValue) => (myValue === '1');
 
-const DayCare = () => {
   // Parameters
   const params = useParams();
-  //const { startDate, endDate, isImmediate, isS117, isFixedPeriod } = params;
+  let {startDate, endDate, isImmediate, isS117, isFixedPeriod} = params;
+  isImmediate = isTrueSet(isImmediate) || false;
+  isS117 = isTrueSet(isS117) || false;
+  isFixedPeriod = checkFixedPeriod(isFixedPeriod) || false;
+  startDate = startDate ?? null;
+  endDate = endDate ?? null;
+
+  const [errors, setErrors] = useState([]);
+  const [termTimeConsiderationOptions, setTermTimeConsiderationOptions] = useState([]);
+  const [opportunitiesLengthOptions, setOpportunitiesLengthOptions] = useState([]);
+  const [opportunityTimesPerMonthOptions, setOpportunityTimesPerMonthOptions] = useState([]);
 
+  const [needToAddress, setNeedToAddress] = useState(undefined);
   const [transportNeeded, setTransportIsNeeded] = useState(undefined);
   const [escortNeeded, setEscortIsNeeded] = useState(undefined);
   const [termTimeConsideration, setTermTimeConsideration] = useState(undefined);
   const [opportunityEntries, setOpportunityEntries] = useState([
-    { id: 1, howLongValue: 45, perMonthValue: 1, needToAddress: undefined },
+    {id: 1, howLongValue: 1, timesPerMonthValue: 1, needToAddress: undefined},
   ]);
+  // Setup days state using base days value
+  const [daysSelected, setDaysSelected] = useState(
+    days.map((dayItem) => {
+      return {...dayItem, checked: false};
+    })
+  );
+
+  useEffect(() => {
+    retrieveTermTimeConsiderationOptions();
+    retrieveOpportunitiesLengthOptions();
+    retrieveOpportunityTimesPerMonthOptions();
+  }, []);
```

Comment:
You may want to pass each of the relevant state values to the array in use effect, so use effect is not triggered on every re-render, within use effect you should then check that each of the corresponding state values is an empty array